api/routes/cartRouter.js

```diff
@@ -0,0 +1,11 @@
+const express = require('express');
+const { cartController } = require('../controllers');
+const { loginRequired } = require('../utils/auth');
+
+const cartRouter = express.Router()
+
+cartRouter.get('', loginRequired, cartController.getCartList);
+cartRouter.post('/:itemId', loginRequired, cartController.addCart);
+cartRouter.delete('/delete', loginRequired, cartController.deleteCart);
```

Comment:
- restful에 맞게 URI에는 동사를 사용하지 않고, HTTP method로 행위를 나타냅니다.
```suggestion
cartRouter.delete('/, loginRequired, cartController.deleteCart);
```